src/SprykerEco/Service/ComputopApi/Mapper/ComputopApiMapper.php

```diff
@@ -134,8 +134,8 @@ public function getDescriptionValue(array $items): string
     public function generateReqIdFromQuoteTransfer(QuoteTransfer $quoteTransfer): string
     {
         return $this->generateReqId(
-            $quoteTransfer->getTotals()->getHash(),
-            $quoteTransfer->getCustomer()->getCustomerReference() ?? static::GUEST_CUSTOMER_REFERENCE
+            $quoteTransfer->getTotalsOrFail()->getHashOrFail(),
```

Comment:
```suggestion
            $quoteTransfer->getTotalsOrFail()->getHash(),
```

hash was not required before
please use `(string)` hack here
don't forget to update API spec

Reply:
Fixed